generate-yaml/README.md

```diff
@@ -0,0 +1,110 @@
+[![New Relic Experimental header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Experimental.png)](https://opensource.newrelic.com/oss-category/#new-relic-experimental)
+
+# 🛠️ Generate YAML changelog action
+
+An action to run the release toolkit generate yaml command. It builds a changelog.yaml file from multiple sources.
+
+## Example Usage
+
+Example generating a changelog yaml from all sources excluding commits whose changes only impact files in `charts` dir:
+```yaml
+- name: Test generate changelog yaml
+  uses: newrelic/release-toolkit/generate-yaml@v1
+  with:
+    excluded-dirs: charts
+```
+
+## Parameters
+
+### Changes
```

Comment:
@alvarocabanas  I've moved the more detailed section in `README.md` here. I think this is the place where we can elaborate all the nitty-gritty details: Which headers are allowed and which aren't, etc.